### PR TITLE
Change Suzuka naming to Movement, m1 to movement-celestia

### LINKED ISSUE #848: Rename suzuka-full-node
Background:
We want to remove reference to "suzuka" on suzuka-full-node as we have testnets with different names and infra partners are getting confused when deploying on Bardock testnet.

Options:
1) Rename suzuka-full-node to testnet-full-node
2) Rename suzuka-full-node to full-node
2) Make a copy of suzuka-full-node and rename to bardock-full-node

Priority: Urgent (P0)

### PR DESCRIPTION
**Pull request by mzabaluev**
Resolves #848, #847.

# Summary
- **Categories**: `networks`, `scripts`, `util`, `cicd`, `misc`.

Rename most of suzuka branding in sources, documentation, scripts, manifests and elsewhere.
The name of the network is Movement.
In particular, rename suzuka-client to movement-tests because everything in that crate is about testing.
This goes towards resolving #417.

# Changelog

Renamed `suzuka-*` binaries and crates to `movement-*`.
Renamed `m1-da-light-node*` binaries and crates to `movement-celestia-da-light-node*`.

# Testing

Existing CI tests should pass (with trigger labels renamed acccordingly).

# Outstanding issues

While we are at it, it would to be good to rename the DA light node as per #847.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -5,7 +5,7 @@
 protocol-units/execution/* @l-monninger @0xmovses @musitdev @mzabaluev
 
 # m1-da
-protocol-units/da/m1/* @l-monninger @mzabaluev
+protocol-units/da/celestia/* @l-monninger @mzabaluev
```
Comment (l-monninger) [inline CODEOWNERS:8]: Ah, I wouldn't do this. Keep it named as the generic abstraction it is. If you need another name from business, ask @rolandoesparza . I would suggest `movement-da` for simplicity.

**Review by l-monninger — CHANGES_REQUESTED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (mzabaluev) [reply CODEOWNERS:8]: There's now `da/grpc` that covers the gRPC related components and this is not specific to a DA layer.
I thought of `da/celestia` as the specific implementation of the DA light node service that works with Celestia. Is there something wrong with this layout and naming?

Comment (l-monninger) [reply CODEOWNERS]: The abstractions in those protocol units are not purely Celestia though. If anything, I would argue that `/celestia` should be a subdir of those protocol units.

**Comment by l-monninger** [comment]
I think this looks good. So, we'll just wait on #873 I guess.

File: CODEOWNERS (hunk 1 above)
Comment (l-monninger) [reply CODEOWNERS]: Sorry, I misread https://github.com/movementlabsxyz/movement/pull/863/commits/320d74b86076b66271e1cac14e96f8974f735b16

I would go for `da/movement`. If you want to have subdirs for just the `grpc` units then `da/movement/grpc`, but I would say `grpc` is providing the abstracted protocol. 

For Celestia, I would either leave it as is, i.e., just being distinguished by structs and modules within the crates, or I would entertain a refactor to move the Celestia implementations of various traits out into their own crates. In this latter case, `da/movement/celestia` would make sense to me.

**Review by l-monninger — CHANGES_REQUESTED** [review]
Still needs fixes.

**Comment by mzabaluev** [comment]
> Still needs fixes.

Pushed the fix in [0f2422b](https://github.com/movementlabsxyz/movement/pull/863/commits/0f2422b6fc7f1b7d6fee61477e5baa8f729d4769)

File: CODEOWNERS (hunk 1 above)
Comment (mzabaluev) [reply CODEOWNERS]: Ah, I see clearer now.

Do we consider Celestia as _the_ DA selected for Movement (the network as in `networks/movement`), or could there be other DAs that we migrate to in the future without hard a hard fork?

Is it conceivable that we could reuse the Celestia DA protocol units for a hypothetical future network that is not named Movement? Then I'd say `da/celestia` in the file tree and`movement-celestia-da*` in crate names (these are global, so we can't just publish `celestia-da*` crates without people getting confused) make more sense.

Comment (l-monninger) [reply CODEOWNERS]: Yes, there could be other DAs. Yes, Celestia DA logic itself might be reused with other components besides this initial `movement` DA.

Comment (mzabaluev) [reply CODEOWNERS]: `da/grpc` might be a poor choice for the subdir I mean to contain abstract protocol glue. Maybe `da/protocol`?

Comment (l-monninger) [reply CODEOWNERS]: This DA is specifically the "Movement DA" as the before was the "M1 DA." SO I would again prefer, `da/movement/protcol` and similar.

Comment (mzabaluev) [reply CODEOWNERS]: Got you, will rearrange accordingly. So I think there should be `da/movement/celestia/*` crates and `da/movement/protocol/light-node`, but then `movement` is the only thing under `da` so it looks like we are future-proofing.

Comment (mzabaluev) [reply CODEOWNERS]: Done in [23d33dc](https://github.com/movementlabsxyz/movement/pull/863/commits/23d33dca6570624b8fe11ddc99c1e49bfa44cfeb)

**Review by l-monninger — APPROVED** [review]

**Pull request merged**